.github/workflows/ci.yml

```diff
@@ -19,7 +19,7 @@ jobs:
       fail-fast: false
       matrix:
         os: [ ubuntu-latest, macos-latest, windows-latest ]
-        python-version: [ "3.9" , "3.10.6" ]
+        python-version: [ "3.9" , "3.10", "3.11" ]
```

Comment:
Python 3.11? That's brave!

Not a bad idea to try with the latest version of Python, but just don't be surprised if things don't work yet. It'll probably be months (years?) before all the common packages actually work with v3.11.

Reply:
Ahh right, I think this is maybe a coming from Julia thing where the updates have often fixed major problems with the language, so people switch pretty quickly. I guess we can keep 3.11 in the CI for now and then remove it if/when it becomes a problem?

Reply:
I briefly added `3.11-dev`  (7c9ef442074040e78be5183e7a9dd5dd74311eb8) and the wheels came off _alarmingly_ quickly.

Reply:
Sure, up to you. It's not a bad thing, but it just might mean you have to remove it from the CI again later -- not a big deal.